app/views/foods/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= link_to new_food_url, :class => "btn btn-dark bg-dark add mt-3 mb-2"  do %>
+Add Food
+<% end %>
+<table class="table table-bordered table-striped">
+  <thead>
+    <tr>
+      <th scope="col">Food</th>
+      <th scope="col">Measurement Unit</th>
+      <th scope="col">Unit Price</th>
+      <th scope="col">Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @foods.each do | food| %>
+    <tr>
+      <th scope="row">
+        <%= link_to food_url(food.id) do %>
+          <%= food.name %>
+        <% end %>
+      </th>
+      <td> <%= food.measurementUnit %> </td>
+      <td> $ <%= food.Price %></td>
+      <td>
+        <%= link_to "Delete Food", food_path(food.id), method: :delete, data: {confirm: "Are you sure"}, :class => "btn btn-sm btn-danger"%>
+      </td>
```

Comment:
- [ ] When I click on the `Delete Food` button it takes me to the food show page instead of deleting the food, then I have to click on the `Delete Food` button again on the food show page before the food will get deleted. Kindly make it such that when a user clicks on the `Delete Food` button in the food index page, the food gets deleted while staying on the food index page.

Reply:
Thanks for your feedback. We fixed the issue.